### PR TITLE
Thayne josh hw7

### PR DESCRIPTION
**Pull request by jdthayne**
I finished camelCase and x-o. For camelCase, I used a regex expression to capitalize the first letter of every word (except for the first word), and then another regex expression to remove the spaces.

For x-o, I split the string on x's and o's and compared the counts of the arrays - 1

### CODE REVIEW

File: hw7/_submissions/thayne-josh/x-o/src/x-o.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const isXAndOValid = str => {
+  let xCount = str.split('x').length-1
+  let oCount = str.split('o').length-1
```
Comment (patrik-drean) [inline hw7/_submissions/thayne-josh/x-o/src/x-o.js]: I love this! One of the best solutions I've seen

File: hw7/_submissions/thayne-josh/x-o/src/x-o.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  if (oCount == xCount) {
+    return true
+  }
+  return false
```
Comment (patrik-drean) [inline hw7/_submissions/thayne-josh/x-o/src/x-o.js]: You can simplify this piece by just doing this:
`return (xCount === oCount)`

File: hw7/_submissions/thayne-josh/camel-case/src/camel-case.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const convertToCamelCase = str => {
+  for(let i = 0; i < str.length; i++) {
+    return str.replace(/(?:^\w|[A-Z]|\b\w)/g, function(word, index) 
+    { 
+        return index == 0 ? word.toLowerCase() : word.toUpperCase(); 
+    }).replace(/\s+/g, ''); 
+  }
```
Comment (patrik-drean) [inline hw7/_submissions/thayne-josh/camel-case/src/camel-case.js]: Some crazy regex 😂 I'm assuming #ThanksGoogle??

Comment (jdthayne) [reply hw7/_submissions/thayne-josh/camel-case/src/camel-case.js]: Haha yeah...

File: hw7/_submissions/thayne-josh/x-o/src/x-o.js (hunk 2 above)
Comment (jdthayne) [reply hw7/_submissions/thayne-josh/x-o/src/x-o.js]: Ohhh duh, thanks!

**Pull request merged**